mcux/mcux-sdk/drivers/flexcomm/fsl_i2c.c

```diff
@@ -887,7 +897,7 @@ status_t I2C_MasterTransferNonBlocking(I2C_Type *base, i2c_master_handle_t *hand
     result = I2C_InitTransferStateMachine(base, handle, xfer);
 
     /* Clear error flags. */
-    I2C_ClearStatusFlags(base, I2C_STAT_MSTARBLOSS_MASK | I2C_STAT_MSTSTSTPERR_MASK);
+    I2C_ClearStatusFlags(base, (uint32_t)((uint32_t)kI2C_MasterAllClearFlags | (uint32_t)kI2C_CommonAllClearFlags));
```

Comment:
```suggestion
    I2C_ClearStatusFlags(base, (uint32_t)(kI2C_MasterAllClearFlags | kI2C_CommonAllClearFlags));
```

Reply:
@hakehuang, this is coming directly from the internal SDK. Can you take this up internally with the SDK team?